basic-data-types/runner-up.py

```diff
@@ -0,0 +1,10 @@
+# Find the Runner-Up Score!
+"""Task
+Given the participants' score sheet for your University Sports Day, you are required to find the runner-up score.
+You are given n scores. Store them in a list and find the score of the runner-up.
+"""
+
+if __name__ == '__main__':
+    n = int(input())
+    print(sorted(list(set(map(int, input().split()))))[-2])
```

Comment:
I won't challenge your approach to go for list->set->list here, because expected input is rather small. However, for bigger collections it's possible to find better algorithm.